### PR TITLE
Add new description field for status update

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward: https://github.com/giantswarm/giantswarm/issues/5751

After this, I would need to update `chartstatus` resource.

### CODE REVIEW

File: helmclient.go [hunk 1]
```diff
@@ -460,6 +460,7 @@ func (c *Client) GetReleaseHistory(ctx context.Context, releaseName string) (*Re
 
 		history = &ReleaseHistory{
 			AppVersion:   appVersion,
+			Description:  release.Info.Description,
```
Comment (rossf7) [inline helmclient.go]: <3

File: types.go [hunk 2]
```diff
@@ -22,6 +22,8 @@ type ReleaseContent struct {
 type ReleaseHistory struct {
 	// AppVersion is the app version of the Helm Chart that has been deployed.
 	AppVersion string
+	// Description is the incident log about past Helm chart deployment issue.
```
Comment (rossf7) [inline types.go]: Is incident log helm terminology? This isn't correct e.g. if the helm release has been upgraded the text will be `Upgrade complete`.

This should describe what information is provided by the tiller API. Not how we want to use it in the operator.

Comment (tomahawk28) [reply types.go]: ```	// Description is the log message about past Helm chart deployment provided by tiller API.``` WDYT?

File: types.go [hunk 3]
```diff
@@ -22,6 +22,8 @@ type ReleaseContent struct {
 type ReleaseHistory struct {
 	// AppVersion is the app version of the Helm Chart that has been deployed.
 	AppVersion string
+	// Description is the log message about past Helm chart deployment provided by tiller API.
```
Comment (rossf7) [inline types.go]:
```suggestion
	// Description is a human-friendly "log entry" about this Helm release.
```

Let's use this based on what they have in upstream.

https://github.com/helm/helm/blob/ae17f3199e519126a83409d08d534d12dadd75aa/_proto/hapi/release/info.proto#L35

Comment (tomahawk28) [reply types.go]: Much simple 😆

**Pull request merged**